src/reference/sectionB_elements.inc

```diff
@@ -454,6 +454,8 @@ A :code:`component_ref` element information item (referred to in this specificat
 
       The value of the :code:`component` attribute MUST be a valid component reference, as defined in the section :ref:`Component references<specC_component_reference>`.
 
+      The value of the :code:`component` attribute MUST NOT be identical to the value of the :code:`component` attribute on any other :code:`component_ref` element in the :ref:`CellML infoset<specA_cellml_infoset>`.
```

Comment:
I thought that there was a reason for not having statements about value in Chapter 2?  We have all the other regulations about interpreting the value of an attribute in Chapter 3? (eg: names of units, names of connected variables, value of component_1 attributes etc etc)?

Reply:
Ones like this can be checked purely syntactically, and we have them in 2. E.g. the uniqueness of component and variable names

Reply:
Search for "MUST NOT be identical"

Reply:
Statements that go in 3 should be ones that you need an interpretation for. So if you can't (easily) do it with pure syntax rules, but need a bit of interpretation to make sense of the rule (e.g. a digraph or network or whatever), then it goes in 3

Reply:
I don't get it ... so why can't we say variable_1 and variable_2 in a mapping can't be equal? or should we be?

Reply:
Some stuff could go in either. For variable_1 and variable_2 I wouldn't have minded if we did some of the stuff syntactically, instead of doing it all through the network definition, but I guess that's a preference?

Reply:
I did think about putting this one in 3 instead. But the similarity to the name uniqueness made me stick it here.
(Also the stuff in 3 starts with "a node for every component in the model", so then the rule would have to be "each node can only correspond to a single component_ref element" or something... so probably better then to rewrite more... This seemed like the minimal effort way to do it!

Reply:
Method in the madness but not madness in the method.  As long as there's a reason, I'm ok.